docs/source/tts/datasets.rst

```diff
@@ -1,10 +1,113 @@
-TTS Datasets
-============
+Data Preprocessing
+==================
 
-Most TTS scripts work out-of-the-box with the LJ Speech dataset.
-However, some models may require supplementary data for training and validation.
-The following sections contain details of required data formats and instructions for running preprocessing scripts for
-such models.
+NeMo TTS recipes support most of public TTS datasets that consist of multiple languages, multiple emotions, and multiple speakers. Current recipes covered English (en-US), German (de-DE), Spanish (es-ES), and Mandarin Chinese (work in progress), while the support for many other languages is under planning. NeMo provides corpus-specific data preprocessing scripts, as shown in the directory of `scripts/data_processing/tts/ <https://github.com/NVIDIA/NeMo/tree/main/scripts/dataset_processing/tts/>`_, to convert common public TTS datasets into the format expected by the dataloaders as defined in `nemo/collections/tts/torch/data.py <https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/tts/torch/data.py>`_. Following the instructions on how to run the corpus-specific scripts below, you can get started with either directly processing those public datasets, or creating your custom scripts to preprocess your custom datasets scripts.
```

Comment:
how about describing the manifest format here? an entry example?

Reply:
good point! will add.